algebra-benches/src/macros/batch_arith.rs

```diff
@@ -0,0 +1,87 @@
+macro_rules! batch_arith {
```

Comment:
Doesn't this run only on nightly?
If yes, let's move these to a criterion benchmark.

Reply:
Agreed to do on another PR.